hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -2713,6 +2713,8 @@ public void testClusteringCommitInPresenceOfInflightCommit() throws Exception {
     // Schedule and execute a clustering plan on the same partition. During conflict resolution the commit should fail.
     HoodieClusteringConfig clusteringConfig = HoodieClusteringConfig.newBuilder().withClusteringMaxNumGroups(10)
         .withClusteringTargetPartitions(0).withInlineClusteringNumCommits(1)
+        .withClusteringUpdatesStrategy("org.apache.hudi.client.clustering.update.strategy.SparkAllowUpdateStrategy")
+        .withRollbackPendingClustering(true)
```

Comment:
this change can be ignored.. i was trying to check whether explicitly setting rollback pending clustering reverts the inflight replacecommit or not. But, this config does not have any effect.